Main.java

```diff
@@ -1,33 +1,37 @@
-package com.company;
-
 import java.util.Scanner;
 
 public class Main {
 
     public static void main(String[] args) {
-        Main.calculatorInteger();
+        System.out.println("Hello, i'm Integer Main");
+        Main main = new Main();
```

Comment:
Create please class `Calculator` - it will contain all calculation logic
when main will contain input logic and will provide received data to the calculator